crates/rethnet_evm_napi/src/context.rs

```diff
@@ -88,19 +97,23 @@ impl Context {
             .enable_time()
             .build()?;
 
-        let hash_generator = Arc::new(Mutex::new(RandomHashGenerator::with_seed("seed")));
+        let mix_hash_generator = Arc::new(Mutex::new(RandomHashGenerator::with_seed(
+            "randomMixHashSeed",
+        )));
+        let state_root_generator = Arc::new(Mutex::new(RandomHashGenerator::with_seed("seed")));
 
         Ok(Self {
             runtime: Arc::new(runtime),
-            hash_generator,
+            mix_hash_generator,
+            state_root_generator,
             #[cfg(feature = "tracing")]
             _tracing_write_guard: guard,
         })
     }
 
     /// Retrieves the context's hash generator.
     pub fn hash_generator(&self) -> &Arc<Mutex<RandomHashGenerator>> {
```

Comment:
should this be renamed to `state_root_generator` now that there are two different hash generators internally?

Reply:
Addressed